### DISCUSSION TITLE
Why my method signature is not marked as covered?

### DISCUSSION THREAD

**Discussion by ws1088** [pytest-dev/pytest#8380] [Q&A]
Hi, 
When I run code coverage with `pytest --cov=xxx` it shows that the lines inside my class method is covered but not the method signature line:
![image](https://user-images.githubusercontent.com/3947543/109431473-ac360a00-79bb-11eb-8dc4-1355417ef9be.png)

Same for the class definition line and the import lines on the top of the file:
![image](https://user-images.githubusercontent.com/3947543/109431534-f8814a00-79bb-11eb-83c2-0a9a845aa8cb.png)

Is there an option to turn this on?
Thanks!

**Comment by ws1088** [comment]
oops, just realize I should have posted to `pytest-cov` [project](https://github.com/pytest-dev/pytest-cov/issues/455)
mod, please removed if needed

**Reply by RonnyPfannschmidt** [reply]
unfortunately pytest-cov doesn't have discussions enabled, else i could move
